### ISSUE TITLE
Underlining text in a paragraph title issue

### ISSUE THREAD

**Issue by jwbos** [IACR/latex#223]
During testing various LaTeX sources I found an issue with iacrcc. 
A minimum working example is:

```
\documentclass{iacrcc}
\begin{document}
\section{Introduction}
\paragraph{\underline{xxx}}Paragraph text
\end{document}
```
(This works with the article class.)
This is due to the usage of the CTAN package [sectsty](https://mirror.koddos.net/CTAN/macros/latex/contrib/sectsty/sectsty.pdf) in iacrcc. This package redefines how (sub)section, chapters and paragraphs should look like and redefines underline in these title environments. We have copied this over from the Transactions. Currently we have:

```
\RequirePackage{sectsty}
\allsectionsfont{\sffamily\boldmath}
```

Is this a real issue? Do we want authors to be able to change the look and feel of (sub)section / (sub)paragraphs?

**Comment by kmccurley** [comment]
I would argue that we should remove the dependence on `sectsty`. It had other problems as I remember (incompatible with `titlesec` which for some reason is used by quite a few IACR authors). The only effect it has for us is to change the font in section headings to sans-serif:
![image](https://github.com/IACR/latex/assets/14105363/2a1e26b1-2e67-4897-b2ba-c71cd9468d02) 
![image](https://github.com/IACR/latex/assets/14105363/16c3cd86-d2bc-49b0-b3c0-ae685f260f78). This is a purely stylistic choice. None of AMS, IEEE, ACM, or Elsevier use this and I would advocate to drop it.

**Comment by jwbos** [comment]
Agreed. Fixed in a0205e39183f216c4f466132298112ab7eef1797. I created a pull request.
